.github/workflows/ncbench.yml

```diff
@@ -0,0 +1,116 @@
+          echo "FREEBAYES_VERSION=${FREEBAYES_VERSION}" >> $GITHUB_ENV
+
+          HAPLOTYPECALLER_VERSION=$(awk '/- bioconda::gatk4=/ {gsub(/.*=/, "", $NF); gsub(/'\''/, "", $NF); print $NF}' modules/nf-core/gatk4/haplotypecaller/environment.yml)
+          echo "HAPLOTYPECALLER_VERSION=${HAPLOTYPECALLER_VERSION}" >> $GITHUB_ENV
+
+          STRELKA_VERSION=$(awk '/- bioconda::strelka=/ {gsub(/.*=/, "", $NF); gsub(/'\''/, "", $NF); print $NF}' modules/nf-core/strelka/germline/environment.yml)
+          echo "STRELKA_VERSION=${STRELKA_VERSION}" >> $GITHUB_ENV
+
+      - name: Download files from AWS
+        uses: keithweaver/aws-s3-github-action@v1.0.0
+        if: ${{ matrix.enabled }}
+        with:
+          # TODO: fix path to agilent test path
+          source: "s3://nf-core-awsmegatests/sarek/results-4bfb25049fdd2bf31c751bafcb6a18eddebcc1d5/test_full_germline_ncbench_agilent/variant_calling" #"s3://${{ secrets.AWS_S3_BUCKET }}/sarek/results-${{ github.sha }}/test_full_germline_ncbench_agilent/variant_calling"
+          command: sync
+          aws_region: eu-west-1
+          destination: ./variant_calling
+          flags: --no-sign-request --include ".vcf.gz" --exclude "g.vcf.gz"
+
+      - name: Install requests
+        run: pip install requests
+
+      - name: Upload to zenodo
+        id: upload_zenodo
+        env:
+          ACCESS_TOKEN: ${{ secrets.NCBENCH_CI_TOKEN_SANDBOX }} # ${{ secrets.ZENODO_DEPOSIT }}
+          PIPELINE_VERSION: ${{ env.PIPELINE_VERSION }}
+        run: python .github/workflows/upload.py
+
+      - name: Store Zenodo deposition ID
+        run: |
+          echo "DEPOSITION_ID=$(cat deposition_id.txt)" >> $GITHUB_ENV
+
+      # update ncbench config yml
+      - name: NCBench set config
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.NCBENCH_CI_TOKEN }}
+          repository: FriederikeHanssen/ncbench-workflow
+          path: ncbench-workflow/
+          ref: main
+
+      - name: Install yq
+        uses: mikefarah/yq@master
+
+      - name: Add entry to config
+        env:
+          PIPELINE_VERSION_NO_DOTS: ${{ env.PIPELINE_VERSION_NO_DOTS}}
+          PIPELINE_VERSION: ${{ env.PIPELINE_VERSION }}
+          FASTP_VERSION: ${{ env.FASTP_VERSION }}
+          BWA_VERSION: ${{ env.BWA_VERSION }}
+          BQSR_VERSION: ${{ env.BQSR_VERSION }}
+          DEEPVARIANT_VERSION: ${{ env.DEEPVARIANT_VERSION }}
+          FREEBAYES_VERSION: ${{ env.FREEBAYES_VERSION }}
+          HAPLOTYPECALLER_VERSION: ${{ env.HAPLOTYPECALLER_VERSION }}
+          STRELKA_VERSION: ${{ env.STRELKA_VERSION }}
+          DEPOSITION_ID: ${{ env.DEPOSITION_ID }}
+        run: bash .github/workflows/set_ncbench_config.sh
+
+      - name: Commit files
+        run: |
+          cd ncbench-workflow
+          git config --global user.email "friederike.hanssen@qbic.uni-tuebingen.de"
+          git config --global user.name "FriederikeHanssen"
```

Comment:
This makes @nf-core-bot sad.

Reply:
:/ any idea how I can do this differently. Or should we fork ncbench to nf-core?